assets/models/system/Mistral-7B-Instruct-v0-2/spec.yaml

```diff
@@ -0,0 +1,32 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: mistralai-Mistral-7B-Instruct-v0-2
+path: ./
+properties:
+  SharedComputeCapacityEnabled: true
+  SHA: 41b61a33a2483885c981aa79e0df6b32407ed873
+  inference-min-sku-spec: 12|1|220|64
+  inference-recommended-sku: Standard_NC12s_v3, Standard_NC24s_v3, Standard_ND40rs_v2, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96amsr_A100_v4, Standard_ND96asr_v4
+  languages: EN
+tags:
+  Featured: ""
+  Preview: ""
+  SharedComputeCapacityEnabled: ""
+  disable-batch: "true"
+  inference_compute_allow_list:
+    [
+      Standard_NC12s_v3,
+      Standard_NC24s_v3,
+      Standard_ND40rs_v2,
+      Standard_NC24ads_A100_v4,
+      Standard_NC48ads_A100_v4,
+      Standard_NC96ads_A100_v4,
+      Standard_ND96amsr_A100_v4,
+      Standard_ND96asr_v4,
+    ]
+  inference_supported_envs:
+    - vllm
+    - ds_mii
```

Comment:
was ds_mii tested? If not please remove it from the list

Reply:
Removed